doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -478,7 +523,28 @@ Multiprotocol Service Layer libraries
 Libraries for networking
 ------------------------
 
-|no_changes_yet_note|
+* Added the :ref:`lib_downloader` library.
+
+* :ref:`lib_download_client` library:
+
+  * Deprecated the :ref:`lib_download_client` library. The application should instead use the :ref:`lib_downloader` library. See migration guide for changes required.
```

Comment:
```suggestion
* Added the :ref:`lib_downloader` library to be used instead of the deprecated :ref:`lib_download_client` library. 
  See the :ref:`migration guide <migration_2.9_recommended>` for required changes.
```